### PR TITLE
Update dependences 

### PR DESCRIPTION
**Pull request by DigitLib**
Add dependence on Islandora Solr Search for OCR module

**JIRA Ticket**: (link)

* https://jira.duraspace.org/browse/ISLANDORA-1564

# What does this Pull Request do?

Add Islandora Solr Search as a dependence 

# What's new?
A in-depth description of the changes made by this PR. Technical details and possible side effects.

Example:
* Changes x feature to such that y
* Added x
* Removed y

# How should this be tested?

A description of what steps someone could take to:
* Reproduce the problem you are fixing (if applicable)
* Test that the Pull Request does what is intended.
* Please be as detailed as possible.
* Good testing instructions help get your PR completed faster.

# Additional Notes:
Any additional information that you think would be helpful when reviewing this PR.

Example:
* Does this change require documentation to be updated? 
* Does this change add any new dependencies? 
* Does this change require any other modifications to be made to the repository (ie. Regeneration activity, etc.)? 
* Could this change impact execution of existing code?

# Interested parties
Tag (@ mention) interested parties or, if unsure, @Islandora/7-x-1-x-committers

### CODE REVIEW

**Review by DiegoPino — APPROVED** [review]
Good. We need a 7.x-1.11 pull too!! Thanks

**Comment by DigitLib** [comment]
@DiegoPino 7.x-1.11 pull added

**Comment by willtp87** [comment]
@DigitLib DigitLib do you have a CLA? https://github.com/Islandora/islandora/wiki/Contributor-License-Agreements

File: README.md
Comment (willtp87) [inline README.md:20]: I don't think Solr is a dependency.  It should probably be documented something along the lines of `Further functionality will be enabled if  _module_ is enabled.`.

Comment (DigitLib) [reply README.md:20]: According to https://jira.duraspace.org/browse/ISLANDORA-1564 ticket it was suggested me to put this as hard dependency.

**Comment by DigitLib** [comment]
@willtp87 No I dont't have a CLA but would like to have...

File: README.md
Comment (willtp87) [reply README.md:20]: @DigitLib That is not how I am reading the comments on that ticket.  If there is confusion perhaps we can discus it at the committers call.

**Comment by willtp87** [comment]
https://github.com/Islandora/islandora_ocr/pull/87#issuecomment-390954743 @dannylamb  commits have been merged, is there anything we need to do here?

File: README.md
Comment (DigitLib) [reply README.md:20]: @willtp87 Ok I wold like to clear this issue also. Waiting for call.

**Comment by DiegoPino** [comment]
@willtp87 @DigitLib long time without having a merge without a CLA, not good but we should be able to work around this somehow, worst scenario reverting (still the history would be maintained so last resource). At least it is a README change so i feel we can work around this. 
Let's start by sending that CLA, shall we?.

@DigitLib (andrija) can you read this https://github.com/Islandora/islandora/wiki/Contributor-License-Agreements and Fill http://islandora.ca/sites/default/files/islandora_cla.pdf (personal contributor agreement) or the corporate, not sure in your case (the personal is always handy i have both) and send it to community@islandora.ca ASAP? @dannylamb will get that one and give you his blessing.

On the topic of this is what was requested or not, the semantics of either version is subtly different, and to be honest by 2018, islandora without Solr is like growing plants without sun. Still, if a big deal lets discuss it during this Thursday's call, will add it to the agenda. 

Thanks folks and thanks @willtp87 for catching the CLA thing.

@Islandora/7-x-1-x-committers please never forget to ask (or review the list) for a CLA to avoid dooms-day situations.

**Comment by DigitLib** [comment]
@DiegoPino thanks! I filled it and send it to community@islandora.ca and thanks @willtp87 for CLA! To be a contributor officially :)
For thursday call will it be possible to start at 1PM EST?

**Comment by DiegoPino** [comment]
@DigitLib should be possible. Will ask in IRC and let's see what folks feel. IF 👍  I will send a google groups post announcing hour change. Can't promise but can try.

**Comment by DigitLib** [comment]
@DiegoPino Great TNX!!!

**Comment by DiegoPino** [comment]
@DigitLib ok, 1PM it is (today). I will add this to the agenda too. Thanks to both

**Comment by DigitLib** [comment]
@DiegoPino GREAT!!!! Tnx! I sow an agenda at google-group :+1:

**Pull request merged**
